NEW_CONNECTION.md

```diff
@@ -0,0 +1,80 @@
+# Add New Data Source
+
+The following are the instructions for adding a new data source 
+
+## Add the following files for Creating a new Datasource
+
```

Comment:
I'd format this as a list and perhaps have two sections: one for new files and another for updated files.